frontend/uddhar-frontend/src/shared/components/Navbar.jsx

```
@@ -48,45 +47,92 @@ const Navbar = ({children}) => {
-                      <nav className="p-2 text-gray-700 hover:bg-blue-200" onClick={()=> navigate(`/dashboard/${user.role}`)}>Dashboard</nav>
-                      <nav className="p-2 text-gray-700 hover:bg-blue-200">Settings</nav>
-                      <nav className="p-2 text-gray-700 hover:bg-blue-200" onClick={logout}>Logout</nav>
-                    </div>
+                    <ul className="absolute right-0 mt-2 w-40 bg-white border rounded-lg z-10 list-none flex flex-col cursor-pointer">
+                      <li className="p-2 text-gray-700 hover:bg-blue-200">
+                        Profile
+                      </li>
+                      <li
+                        className="p-2 text-gray-700 hover:bg-blue-200"
+                        onClick={() => navigate(`/dashboard/${user.role}`)}
+                      >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)